package.json

```diff
@@ -17,8 +17,8 @@
   "type": "module",
   "engineStrict": true,
   "engines": {
-    "node": ">=16",
-    "npm": ">=8"
+    "node": ">=18",
+    "npm": ">=9"
```

Comment:
What is the benefit of constraining the npm version?

Reply:
I only did it for consistency since I saw it in some places but not others.  Should I remove it everywhere?

Reply:
There is _some_ use to having them in the top-level package.json, so that you at least get a warning if you try to build the project with an unsupported version. In other places, yes, let's remove them 🙂

Reply:
sure! I'm happy to put it in whichever ones you think is best :)

Which ones are the top-level packages?  E.g. `connect` didn't previously have an npm designation.  Should it now?

Reply:
[This](https://github.com/bufbuild/connect-es/blob/29dc336bff2246044a9ac331ee70fedcfdcfa46c/package.json#L19-L22) is the top-level package.json. It's only relevant for contributors. 

I do not think the @bufbuild/connect package.json should have an engines field. If you are aware of a good benefit for constraining the npm version there, I'm all ears. If there isn't a really good one, we can save some time by not setting it, and then never ever having to update it.

Reply:
ahhh gotcha gotcha.  I misunderstood because you said "them", so I thought you were talking about more than one.

please just tell me plainly which one's you'd like to have/not have and I'd be happy to update.  It sounds like you're saying to remove all engines fields except the repo-root package.json.